jenkins/jobs/trigger/nightly_trigger.sh

```diff
@@ -24,15 +24,15 @@ JENKINS_JOB=$4
 JENKINS_URL=https://vic-jenkins.eng.vmware.com/
 JENKINS_USER=svc.vicuser
 JENKINS_PASSWD=bx741zG2rMN7G9PuXCh
-SCRIPT_DIR=$(cd $(dirname "$0") && pwd)
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
```

Comment:
I've always used `$(readlink -f "$BASH_SOURCE")` because of reasons that I don't recall other than the internet said it was the least likely to break under edge cases and the detail logic was sound.
Obviously in this case we can control the edge cases, but...

Reply:
I'll switch to:
> `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"`

From https://stackoverflow.com/a/246128